packages/frontend-2/components/viewer/resources/ActiveVersionCard.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="bg-primary-muted flex gap-2 py-2 px-4">
+    <EyeIcon v-tippy="'Currently viewing'" class="h-4 w-4 text-primary mt-0_5" />
+    <div class="flex flex-col overflow-hidden text-xs select-none">
+      <div class="inline-block rounded-full font-bold">
+        {{ version.sourceApplication }}
+      </div>
+      <div class="truncate text-foreground-2">
+        {{ version.message || 'no message' }}
+      </div>
+      <div class="italic text-foreground-2">
+        {{ timeAgo(version.createdAt) }}
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import { EyeIcon } from '@heroicons/vue/24/solid'
+
+interface Version {
```

Comment:
This is good enough for now, but in the future I suggest looking into GQL fragment-oriented prop design: https://the-guild.dev/blog/unleash-the-power-of-fragments-with-graphql-codegen

Essentially, you can define a GQL fragment specifying the fields you're interested in, then use the auto-generated TS type for that fragment in your `defineProps` and also specify that fragment in any queries that load the data for this component, ensuring that even if you change fragment fields, both the GQL query and the component will have these changes.

Reply:
We already have many examples of this in FE2, btw, for example `ProjectModelsPageHeader_Project` in `Header.vue`